tests/10apidoc/00prepare.pl

```diff
@@ -55,17 +56,30 @@ sub assert_room_members {
 
    assert_json_keys( $room, qw( timeline state ephemeral ));
 
-   return assert_room_members_in_state( $room->{state}{events}, $member_ids );
+   return assert_room_members_in_state( $room->{state}{events}, $memberships );
 }
 
 
 # assert that the given members are present in a block of state events
 sub assert_room_members_in_state {
-   my ( $events, $member_ids ) = @_;
+   my ( $events, $memberships ) = @_;
+   # Takes either an arrayref of user_ids or a hashref of user_id to membership strings
 
-   log_if_fail "members:", $member_ids;
+   log_if_fail "members:", $memberships;
    log_if_fail "state:", $events;
 
+   my ( $member_ids );
+   if (ref($memberships) eq 'ARRAY') {
```

Comment:
this (and other stuff below) isn't consistent with the whitespace conventions for sytest. I'm not a fan of those conventions, but I think having mismatches within a file is not a good thing.